wire-ios/Wire-iOS/Sources/Authentication/Event Handlers/Input/AuthenticationLoginCredentialsInputHandler.swift

```diff
@@ -47,7 +47,12 @@ final class AuthenticationLoginCredentialsInputHandler: AuthenticationEventHandl
 
 }
 
-struct EmailPasswordInput {
+struct EmailPasswordInput: CustomStringConvertible {
+
     var email: String
     var password: String
+
+    var description: String {
```

Comment:
just to understand this is not used actually, is it?

Reply:
Due to my change in AuthenticationEventResponderChain.swift currently it shouldn't be used, but this might prevent a future issue when a this type is accidentally converted to a string.